theme/less/content/colors.less

```diff
@@ -124,6 +134,8 @@ each(@colors, {
 
     .bg-@{value} {
         background-color: @@value;
+        --hover-tint: @@value;
```

Comment:
Found an issue with Participate button hover color
![image](https://user-images.githubusercontent.com/28541902/168324315-3d6f2fb3-a738-4359-8978-76ff54302936.png)

Reply:
DSFR changes the way they handle background for secondary button. Now it's a flat grey light color. We'll have to find another way to make it work with our various colors.